ui/src/components/SampleGrid/TaskItem.jsx

```diff
@@ -201,25 +201,25 @@ export class TaskItem extends React.Component {
   stateClass() {
     const task = this.props.taskData;
 
-    let varaint = 'primary';
+    let variant = 'primary';
 
     if (task.state === TASK_RUNNING) {
-      varaint = 'warning';
+      variant = 'warning';
     } if (task.state === TASK_COLLECT_FAILED) {
-      varaint = 'danger';
+      variant = 'danger';
     } if (task.state === TASK_COLLECT_WARNING) {
-      varaint = 'danger';
+      variant = 'danger';
     } else if (task.state === TASK_COLLECTED) {
-      varaint = 'success';
+      variant = 'success';
     }
 
-    return varaint;
+    return variant;
   }
 
   popoverPosition() {
     const viewportHeight = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
     let result = 'bottom';
-
+    debugger;
```

Comment:
Watch out for this debugger statement :)

Reply:
Thanks  a lot Marcus, 
I have to add more more commit to this PR. 

[fix modal display] commit have some side effect on the Characterisation Modal. sorry 
Did not notice.